App.js

```diff
@@ -33,16 +46,25 @@ export default class App extends React.Component {
         });
     };
 
+
     render() {
         return (
-            <View style={styles.container}>
-                <Text h2>Today!</Text>
-                <TodoList items={this.state.list} onCheckItem={this.handleCheckItem}/>
-                <Icon
-                    name='add-circle'
-                    color='#f50'
-                    onPress={() => this.handleAddItem('TODO: pop a modal to capture new item.  also, style this button plz')}/>
-            </View>
+
+            <React.Fragment>
+                {this.state.fontLoaded ? (
```

Comment:
don't load any of our stuff until the font is loaded.  we don't want a flash going on.